
## Code execution
The code provided in sequences_extraction.py allows to extract the aligned sequences from the BLAST results text file and save them in fasta format.
To execute the code use:

python sequence_extraction.py -input *name of the input file* -output *name of the output file*

## Additional data
The dataset used to construct guide tree can be found in guide_tree_dataset.fasta file

The reference tree dataset can be found in annotated_both_tir_clean.fasta file

The dataset containg both complete genes and pseufogenes can be founf in annotated_pseudo.fasta file

The file aa_horizontal.fasta containd a dataset showing horizontal transfer of genes in HZTransib

All identified Terminal Inverted Repeats (TIR) can be found in TIR_search_results.txt file
